### ISSUE TITLE
Woocommerce 'New Product' instead of New Post?

### ISSUE THREAD

**Issue by kamazoy** [maxcutler/python-wordpress-xmlrpc#70]
Hi,

Looking at [this page](http://wordpress.stackexchange.com/questions/137501/how-to-add-product-in-woocommerce-with-php-code) it looks like it's possible to use the XMLRPC endpoint to create/update/etc products in Woocommerce, which itself uses posts of type "product" instead of post/parent/etc - is it possible to use your code to do this?

I have tried to do this with the woocommerce REST API in Python and not gotten too far with it.

Best wishes

Daniel

**Comment by zach-adams** [comment]
Absolutely, just be sure to set the 'post_type' to 'product'

**Comment by kamazoy** [comment]
Yeah this doesn't seem to work, there's a ton of meta that it needs to go along with it.                                                                                                                                                                                                                 From: Zach AdamsSent: Friday, 6 February 2015 05:17To: maxcutler/python-wordpress-xmlrpcReply To: maxcutler/python-wordpress-xmlrpcCc: Daniel DaintySubject: Re: [python-wordpress-xmlrpc] Woocommerce 'New Product' instead of New Post? (#70)Absolutely, just be sure to set the 'post_type' to 'product'

—Reply to this email directly or view it on GitHub.

**Comment by vccabral** [comment]
@kamazoy did you end up getting it working by extended AuthenticatedMethod?

**Comment by WilfredChong** [comment]
I tried 'post_typ' to 'product' and it work, but I cannot add other meta like 'sku', 'description', price etc.
I tried add it in custom_fields but still not working.
